app/models/organization.rb

```diff
@@ -29,14 +29,20 @@
 class Organization < ApplicationRecord
   DIAPER_APP_LOGO = Rails.root.join("public", "img", "diaperbase-logo-full.png")
 
+  MIN_DAY_OF_MONTH = 1
+  MAX_DAY_OF_MONTH = 28
+
   validates :name, presence: true
   validates :short_name, presence: true, format: /\A[a-z0-9_]+\z/i
   validates :url, format: { with: URI::DEFAULT_PARSER.make_regexp, message: "it should look like 'http://www.example.com'" }, allow_blank: true
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_blank: true
   validate :correct_logo_mime_type
-  validates :deadline_day, numericality: { only_integer: true, less_than_or_equal_to: 28, greater_than_or_equal_to: 1, allow_nil: true }
-  validates :reminder_day, numericality: { only_integer: true, less_than_or_equal_to: 14, greater_than_or_equal_to: 1, allow_nil: true }
-  validate :deadline_after_reminder
+  validates :deadline_day, numericality: { only_integer: true, less_than_or_equal_to: MAX_DAY_OF_MONTH,
+                                           greater_than_or_equal_to: MIN_DAY_OF_MONTH, allow_nil: true }
+  validates :reminder_day, numericality: { only_integer: true, less_than_or_equal_to: MAX_DAY_OF_MONTH,
+                                           greater_than_or_equal_to: MIN_DAY_OF_MONTH, allow_nil: true }
+
+  validates :reminder_day, numericality: { other_than: :deadline_day }, if: :deadline_day?
```

Comment:
Shall we make these updates also on the PartnerGroup model?

Reply:
Consistency would be great, but I think it could also be part of a separate issue.

Reply:
Done anyway :-).